specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -25,10 +25,10 @@ For other options on installation see [Installing AutoRest](https://aka.ms/autor
 These are the global settings for the KubernetesConfiguration.
 
 ``` yaml
-title: SourceControlConfigurationClient
+title: KubernetesConfigurationClient
```

Comment:
This should be a breaking change for the existing SDK ?

Reply:
I'll revert the change. Is there a process to introduce a breaking change like this without affecting older SDKs?

Reply:
If you add this line back https://github.com/Azure/azure-rest-api-specs/pull/13374/files#diff-c69f21c5dd5c0d2ae5478d5a35443405ffdc67b95fdb98c374f8daed5893c22bL12 it should be able to work. 
If you add this line to all the existing SDK configuration this line itself doesn't really count as a breaking change. Just may cause inconsistency between newer languages SDKs and existing SDKs.

Reply:
Changing back, will look at this change in a future PR. If you have a process for making this change, can you please point me to the documentation for this? Thanks

Reply:
Actually, this kind of breaking change can't be detected for now. I will discuss with other teammates to see how should we do about this. 
cc @nickzhums

Reply:
If this is not an issue, we would rather take it now compared to having to deal with this later.